src/unity/python/turicreate/toolkits/classifier/logistic_classifier.py

```diff
@@ -621,10 +621,12 @@ class as a vector. The probability of the first class (sorted
         >>> class_predictions = model.predict(data, output_type='class')
 
         """
-
+        
```

Comment:
Let's just revert back to the original `logistic_classifier.py`; I don't think we need this PR to touch this file at all